MANIFEST.in

```diff
@@ -1,9 +1,13 @@
+graft crfsuite
+graft liblbfgs
+graft include
+graft pycrfsuite
+graft tests
+
+include LICENSE.txt
 include README.rst
 include CHANGES.rst
 include update_cpp.sh
-include include/inttypes.h
 
-recursive-include crfsuite *
-recursive-include liblbfgs *
-recursive-include tests *.py
-recursive-include pycrfsuite *.py *.pxd *.pyx *.cpp *.hpp
+global-exclude .py[cod] __pycache__
```

Comment:
I used explicit includes instead of explicit excludes because sometimes there were other files in these folders, e.g. html reports generated by `cython --annotate`, or maybe some temporary files in tests folder which failed to clean up, or IPython notebooks (yeah my folders are not always clean). I guess this is not a problem if releases are done on Travis. But could you please at least add html files to excludes, if we switch to include-by-default?